testing/python/metafunc.py

```diff
@@ -1438,6 +1443,31 @@ def test_limit(limit, myfixture):
         reprec = testdir.inline_run()
         reprec.assertoutcome(passed=2)
 
+
+    @pytest.mark.parametrize('strict', [True, False])
+    def test_parametrize_marked_value(self, testdir, strict):
+        s = """
+            import pytest
+
+            @pytest.mark.parametrize(("n", "expected"), [
+                pytest.param(
+                    2,3,
+                    marks=pytest.mark.xfail("sys.version_info > (0, 0, 0)", reason="some bug", strict={strict}),
+                ),
+                pytest.param(
+                    2,3,
+                    marks=[pytest.mark.xfail("sys.version_info > (0, 0, 0)", reason="some bug", strict={strict})],
+                ),
+            ])
+            def test_increment(n, expected):
+                assert n + 1 == expected
+        """.format(strict=strict)
+        testdir.makepyfile(s)
+        reprec = testdir.inline_run()
+        passed, failed = (0, 2) if strict else (2, 0)
+        reprec.assertoutcome(passed=passed, failed=failed)
+
```

Comment:
We should add a test for the situation where the user uses passes `id` to a `param` and to `parametrize` in the same call.